From 3b0817e1f84226e1b227185706b2b2c62216e292 Mon Sep 17 00:00:00 2001
From: Didier Roche <didrocks@ubuntu.com>
Date: Tue, 22 Jun 2010 12:13:42 +0200
Subject: [PATCH] fix issue in find | xargs when files contain spaces (Closes: #586755)

When files had spaces, find | xargs md5sum was executing md5sum on the
two invalid partial filename, which resuted in md5sum exiting with the
error code 123 and making the build fail.
---
 dh_autoreconf |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/dh_autoreconf b/dh_autoreconf
index 382656a..ad2e317 100755
--- a/dh_autoreconf
+++ b/dh_autoreconf
@@ -70,7 +70,7 @@ if (defined($dh{EXCLUDE_FIND}) && $dh{EXCLUDE_FIND} ne '') {
 }
 
 my %modes = (
-    'md5' => "find $find_options -type f | xargs md5sum",
+    'md5' => "find $find_options -type f -exec md5sum {} \\;",
     'timesize' => "find $find_options -type f -printf \"%s|%T@  %p\n\""
 );
 
-- 
1.7.1

